DNN Platform/DotNetNuke.Abstractions/Portals/IPortalInfo.cs

```diff
@@ -0,0 +1,310 @@
+        /// </summary>
+        /// <value>Currency of the portal.</value>
+        string Currency { get; set; }
+
+        /// <summary>
+        /// Gets or sets default language for the portal.
+        /// </summary>
+        /// <value>Default language of the portal.</value>
+        string DefaultLanguage { get; set; }
+
+        /// <summary>
+        /// Gets or sets description of the portal.
+        /// </summary>
+        /// <value>Description of the portal.</value>
+        string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the default e-mail to be used in the porta.
+        /// </summary>
+        /// <value>E-mail of the portal.</value>
+        string Email { get; set; }
+
+        /// <summary>
+        /// Gets or sets date at which the portal expires.
+        /// </summary>
+        /// <value>Date of expiration of the portal.</value>
+        DateTime ExpiryDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets gUID of the portal info object.
+        /// </summary>
+        /// <value>Portal info Object GUID.</value>
+        Guid GUID { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabdId of the Home page.
+        /// </summary>
+        /// <value>TabId of the Home page.</value>
+        int HomeTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets amount of currency that is used as a hosting fee of the portal.
+        /// </summary>
+        /// <value>Currency amount hosting fee.</value>
+        float HostFee { get; set; }
+
+        /// <summary>
+        /// Gets or sets total disk space allowed for the portal (Mb). 0 means not limited.
+        /// </summary>
+        /// <value>Diskspace allowed for the portal.</value>
+        int HostSpace { get; set; }
+
+        /// <summary>
+        /// Gets or sets keywords (separated by ,) for this portal.
+        /// </summary>
+        /// <value>Keywords seperated by .</value>
+        string KeyWords { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabId with the login control, page to login.
+        /// </summary>
+        /// <value>TabId of the Login page.</value>
+        int LoginTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the logo associated with the portal. The admin can set the logo in the portal settings.
+        /// </summary>
+        /// <value>URL of the logo.</value>
+        string LogoFile { get; set; }
+
+        /// <summary>
+        /// Gets or sets number of portal pages allowed in the portal. 0 means not limited.
+        /// </summary>
+        /// <value>Number of portal pages allowed.</value>
+        int PageQuota { get; set; }
+
+        /// <summary>
+        /// Gets or sets name of the Payment processor that is used for portal payments, e.g. PayPal.
+        /// </summary>
+        /// <value>Name of the portal payment processor.</value>
+        string PaymentProcessor { get; set; }
```

Comment:
@mitchelsellers is this feature deprecated?

Reply:
Oh yeah...I don't think we even have an interface for it

Reply:
Should I remove anything around Payments from `IPortalInfo`?

Reply:
I think I would vote yes.  @bdukes

Reply:
Yeah, at least remove from this interface, perhaps deprecate on `PortalInfo` itself…

Reply:
Just took a look and that would be removing the following properties from the `IPortalInfo`
- `PaymentProcessor`
- `ProcessorPassword`
- `ProcessorUserId`

Reply:
Also `HostFee` and `Currency`

Reply:
Right now the running list is:
- `PaymentProcessor`
- `ProcessorPassword`
- `ProcessorUserId`
- `HostFee`
- `Currency`

Are there any other fields that we are missing?

Reply:
Final list - 

*  BackgroundFile 
*  BannerAdvertising 
*  PaymentProcessor
*  ProcessorPassword
*  ProcessorUserId
*  HostFee
*  Currency
*  Users
*  Version
*  Pages

Reply:
I would keep "Currency", because this might be used by numerous 3rd party extensions

Reply:
I have removed `Currency` as requested in our discussion, but I am leaving this thread open so there could be more discussion. I think marking `Currency` as deprecated will give developers enough time to migrate their code to use `PortalSettings`.